src/main/java/org/gridsuite/modification/server/modifications/TabularModification.java

```diff
@@ -45,13 +45,15 @@ public void check(Network network) throws NetworkModificationException {
     @Override
     public void apply(Network network, Reporter subReporter) {
         AtomicInteger applicationFailuresCount = new AtomicInteger(0);
-        modificationInfos.getModifications().forEach(modification -> {
+        modificationInfos.getModifications().forEach(modificationInfos -> {
```

Comment:
We can probably fix the Sonarcube code smell with little effort.

Reply:
Didn't see it, done